wake-lock/wakelock-type.https.any.js

```diff
@@ -0,0 +1,19 @@
+//META: title=WakeLock.request() with invaild type
+
+promise_test(async t => {
+  await promise_rejects(t, new TypeError(), WakeLock.request());
+}, "'TypeError' is thrown when set an empty wake lock type");
+
+promise_test(async t => {
+  const invalidTypes = [
+    "invalid",
+    null,
+    123,
+    {},
+    "",
+    true
+  ];
+  invalidTypes.map(invalidType => {
```

Comment:
You need to make this function `async` in order to use `await` in the line below.

Reply:
Done, thanks!